app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -172,29 +172,33 @@ def parse_env_variables(env_variables)
       end
 
       def parse_openshift_image(openshift_image)
-        id = openshift_image[:dockerImageReference]
-        ref = ContainerImage::DOCKER_PULLABLE_PREFIX + id
+        id = openshift_image[:dockerImageReference] || openshift_image[:metadata][:name]
+        ref = ContainerImage::DOCKER_PULLABLE_PREFIX + id.to_s
         new_result = parse_container_image(id, ref)
 
-        json = JSON.parse(openshift_image[:dockerImageManifest])
-        new_result[:tag] ||= json["tag"] if json.keys.include?("tag")
+        if openshift_image[:dockerImageManifest].present?
+          json = JSON.parse(openshift_image[:dockerImageManifest])
```

Comment:
Is it possible this could be a "bad" JSON payload?  If so, does it need to be handled?